### PR TITLE
GoldenGate Basics

### PR DESCRIPTION
**Pull request by catlook**
In a comment on the pull request, include the following information and check off the list.

 - [ ] Mini bio
  - Soft capped to 75 words
 - [ ] Desired date of publishing
  - Make sure this matches the `date` in the front matter
 - [ ] Social media contact info
 - [ ] Redaction: (yes|no)
  - Allow redaction, editing or structure changes by technical writer staff
 - [ ] Comments: (yes|no)
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-f204012926/)

File: _posts/2018-08-16-oracle-goldengate-basics.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+date: 2018-08-16 00:00
+comments: true
+author: Tej Singh
+published: true
+authorIsRacker: true
+categories:
+    - Oracle
+---
+
+This blog post explores basics of Oracle&reg; GoldenGate&reg; and its functions.
```
Comment (stephamon) [inline _posts/2018-08-16-oracle-goldengate-basics.md]: ...explores the basics...

File: _posts/2018-08-16-oracle-goldengate-basics.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+Oracle GoldenGate is a software product that allows you to replicate, filter,
+and transform data from one database to another database.
+
+It enables the replication of data between Oracle databases and other supported
+heterogeneous databases.
+
+### Why Use GoldenGate?
+
+The following list highlights some reasons to use Goldegate:
```
Comment (stephamon) [inline _posts/2018-08-16-oracle-goldengate-basics.md]: Some reasons to use GoldenGate include:

File: _posts/2018-08-16-oracle-goldengate-basics.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+-	Data integration
+-	Live reporting
+
+#### How to use GoldenGate
+
+There are many different architectures that can be configured, ranging from a
+simple unidirectional architecture to a more complex peer-to-peer one. Refer to
+the following image to see the various topologies supported by GoldenGate.
+
+![]({% asset_path 2018-08-16-oracle-goldengate-basics/Picture1.png %})
```
Comment (stephamon) [inline _posts/2018-08-16-oracle-goldengate-basics.md]: Is there a source for this image?

File: _posts/2018-08-16-oracle-goldengate-basics.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+
+The manager is the process which starts the other GoldenGate processes. This
+process must run on both the source and target system for the configuration and
+must initiate all the other GoldenGate processes. The manager process also
+allocates disk space by purging the old trail files. One manager process is
+required for every GoldenGate installation.
+
+#### Extract
+
+The extract process involves data extraction, which is called data capture in
```
Comment (stephamon) [inline _posts/2018-08-16-oracle-goldengate-basics.md]: Italicize _data capture_

File: _posts/2018-08-16-oracle-goldengate-basics.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+GoldenGate. Extract is the process that is configured to run against the source
+mining database.
+
+Extract is responsible for capturing the committed data modelling language (DML)
```
Comment (stephamon) [inline _posts/2018-08-16-oracle-goldengate-basics.md]: "modelling" should be "modeling"

File: _posts/2018-08-16-oracle-goldengate-basics.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+applies them to the target database.
+
+#### Data pump
+
+The pump process, which is also an extract process, is optional in the GoldenGate
+setup. This process copies the trail files that contain data to the target system.
+
+#### Checkpoint
+
+The extract pump & replicat processes use checkpoints for tracking their
```
Comment (stephamon) [inline _posts/2018-08-16-oracle-goldengate-basics.md]: the ampersand should just be "and"

File: _posts/2018-08-16-oracle-goldengate-basics.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+respective progress. This mechanism marks the location of data changes that have
+been retrieved or applied from the trail files. This is useful when processes
+need to recover (without any data loss) or need to know the starting point after
```
Comment (stephamon) [inline _posts/2018-08-16-oracle-goldengate-basics.md]: probably you can just say "need to recover without any data loss" without the parenthesis

File: _posts/2018-08-16-oracle-goldengate-basics.md [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+
+IC is a new form of the extract process, which has moved inside the source
+database. In the traditional classic extract process, the extract works on the
+redo logs outside the domain of the actual database. In IC, a server
+**log miner** process is started, which extracts all the DML data and DDL
+statements (create, alter, drop, and so on) and creates Logical Change Records
+(LCR). These records are then handed to the GoldenGate memory processes, which
+write the LCR’s to the local trail files.
+
+### GoldenGate Replicat modes
```
Comment (stephamon) [inline _posts/2018-08-16-oracle-goldengate-basics.md]: GoldenGate replicat modes

File: _posts/2018-08-16-oracle-goldengate-basics.md [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+target database.
+
+The following image illustrates integrated replicat:
+
+![]({% asset_path 2018-08-16-oracle-goldengate-basics/Picture5.png %})
+
+**Image Source**: https://docs.oracle.com/GoldenGate/1212/gg-winux/GIORA/process_mode.htm#GIORA212
+
+
+### Conclusion:
```
Comment (stephamon) [inline _posts/2018-08-16-oracle-goldengate-basics.md]: Remove the colon here

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-f204012926/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-32c169b017/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-c541e8f658/). :bow:

**Comment by cyrichardson** [comment]
@catlook Suggested changes here: 
[blog_Oracle-GoldenGate-basics_cyrEdits.docx](https://github.com/rackerlabs/docs-developer-blog/files/2294708/blog_Oracle-GoldenGate-basics_cyrEdits.docx)

**Pull request merged**